src/main/java/org/stellar/sdk/responses/GsonSingleton.java

```diff
@@ -43,9 +45,24 @@ public static Gson getInstance() {
                       .registerTypeAdapter(tradePageType.getType(), new PageDeserializer<TradeResponse>(tradePageType))
                       .registerTypeAdapter(tradeAggregationPageType.getType(), new PageDeserializer<TradeAggregationResponse>(tradeAggregationPageType))
                       .registerTypeAdapter(transactionPageType.getType(), new PageDeserializer<TransactionResponse>(transactionPageType))
+                      .registerTypeHierarchyAdapter(byte[].class,new ByteArrayToBase64TypeAdapter())
```

Comment:
From Gson [docs](https://google.github.io/gson/apidocs/com/google/gson/GsonBuilder.html#registerTypeHierarchyAdapter-java.lang.Class-java.lang.Object-):
> Configures Gson for custom serialization or deserialization for an inheritance type hierarchy.

Is this the case here. Wouldn't _normal_ `registerTypeAdapter` work here?

Reply:
not sure, just wanted to have a working solution